### PR TITLE
Improve metainfo file

### PR DESCRIPTION
**Pull request by Alexander-Wilms**
The URL types have been used according to https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-url

I've removed the mention of the license from the list of features, since it's usually shown in a separate badge, like in KDE Discover or on `[Flathub](https://flathub.org/apps/org.librepcb.LibrePCB)`:
image

Perhaps the mentioned innovative concepts could be expanded upon.

### CODE REVIEW

File: share/metainfo/org.librepcb.LibrePCB.appdata.xml [hunk 1]
```diff
@@ -46,10 +45,24 @@
...
   <url type="bugtracker">https://github.com/LibrePCB/LibrePCB/issues</url>
+  <url type="faq">https://librepcb.org/faq/</url>
   <url type="help">https://librepcb.org/docs/</url>
-
+  <url type="donation">https://librepcb.org/donate/</url>
+  <url type="translate">https://explore.transifex.com/librepcb/librepcb-application/</url>
+  <url type="contact">https://librepcb.org/contact/</url>
+  <url type="vcs-browser">https://github.com/LibrePCB/LibrePCB</url>
+  <url type="contribute">https://librepcb.org/contribute/</url>
+
```
Comment (ubruhin) [inline share/metainfo/org.librepcb.LibrePCB.appdata.xml:65]: These trailing spaces should be removed to make CI happy :see_no_evil:

**Review by ubruhin — CHANGES_REQUESTED** [review]
Looks great, thanks! Only a small whitespace issue which makes the stylecheck fail.

You can ignore the macOS CI failure, somehow it suddenly stopped working but it's not related to this PR :sob:

**Comment by ubruhin** [comment]
I just saw there's a validation tool for this file and quickly run it:

```
$ appstream-util validate ./share/metainfo/org.librepcb.LibrePCB.metainfo.xml 
./share/metainfo/org.librepcb.LibrePCB.metainfo.xml: FAILED:
• tag-invalid           : <url> type invalid [unknown]
Validation of files failed
```

Any idea what's the exact problem? May it be related to the tool version?

```
$ appstream-util --version
Version:	0.7.18
```

**Comment by Alexander-Wilms** [comment]
Yes, I think these new URL types are not supported by older versions of appstream-util.
Cf. https://bugzilla.redhat.com/show_bug.cgi?id=1830923

The [Flathub wiki](https://github.com/flathub/flathub/wiki/AppData-Guidelines#use-flathubs-appstream-util) suggests using the flatpak'd version:

```
flatpak install flathub org.freedesktop.appstream-glib
flatpak run org.freedesktop.appstream-glib validate share/metainfo/org.librepcb.LibrePCB.metainfo.xml
share/metainfo/org.librepcb.LibrePCB.metainfo.xml: OK
```

**Comment by Alexander-Wilms** [comment]
This set of changes is ready to be merged.

Perhaps newer screenshots showcasing the 1.0 GUI using a 16:9 aspect ratio could be added in a separate PR.

**Pull request merged**